### PR TITLE
update the api conversion tool to default to v1

### PR DESCRIPTION
**Pull request by caesarxuchao**
#7018

@krousey @nikhiljindal @bgrant0607

### CODE REVIEW

File: cmd/kube-version-change/version.go [hunk 1]
```diff
@@ -39,7 +40,7 @@ var (
 	inputSource   = flag.StringP("input", "i", "-", "Input source; '-' means stdin")
 	outputDest    = flag.StringP("output", "o", "-", "Output destination; '-' means stdout")
 	rewrite       = flag.StringP("rewrite", "r", "", "If nonempty, use this as both input and output.")
-	outputVersion = flag.StringP("out-version", "v", "v1beta3", "Version to convert input to")
+	outputVersion = flag.StringP("out-version", "v", "v1", "Version to convert input to")
```
Comment (nikhiljindal) [inline cmd/kube-version-change/version.go]: Can we use latest.Version here?

**Comment by caesarxuchao** [comment]
Thanks @nikhiljindal. PTAL.

File: cmd/kube-version-change/version.go [hunk 2]
```diff
@@ -28,6 +28,8 @@ import (
 	"runtime"
 
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/latest"
+	_ "github.com/GoogleCloudPlatform/kubernetes/pkg/api/v1"
```
Comment (nikhiljindal) [inline cmd/kube-version-change/version.go]: Why is this required?

Comment (caesarxuchao) [inline cmd/kube-version-change/version.go]: To call the init() function in pkg/api/v1beta3/register.go, otherwise api.Scheme will not have v1 registered.

Comment (nikhiljindal) [inline cmd/kube-version-change/version.go]: api/latest already does that.
It also requires that apiVersion/register.go's init() is called.

Comment (caesarxuchao) [inline cmd/kube-version-change/version.go]: Thanks. Yes, now that we add api/latest in the imports, we can remove imports of v1/v1beta3 now.

**Comment by caesarxuchao** [comment]
@nikhiljindal  Updated, PTAL. Thanks.

**Comment by nikhiljindal** [comment]
LGTM, thanks!
